### PR TITLE
[plugin.video.newyankeeworkshop] v0.0.1

### PR DESCRIPTION
**Pull request by th0ma5w**
### Description

This plugin allows for viewing of videos directly from the official website for The New Yankee Workshop at https://www.newyankee.com/ and will automatically update with new videos each month until the archive they are building is complete.
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.newyankeeworkshop/newyankee.py
Comment (Lunatixz) [inline plugin.video.newyankeeworkshop/newyankee.py]: iconImage, thumbnailImage are deprecated, please use setArt https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#gad3f9b9befa5f3d2f4683f9957264dbbe

File: plugin.video.newyankeeworkshop/router.py
Comment (Lunatixz) [inline plugin.video.newyankeeworkshop/router.py]: `iconImage `is deprecated please use setArt.

**Review by Lunatixz — DISMISSED** [review]

**Comment by th0ma5w** [comment]
Okay I believe I corrected all of the changes and rebased everything back to 1 commit. Feel free to let me know about any other possible changes needed or improvements and I'll be happy to update it. Thanks again!

**Comment by Lunatixz** [comment]
LGTM, Thanks

**Comment by Lunatixz** [comment]
@Rechi @enen92 needs second review

**Comment by th0ma5w** [comment]
@Rechi @Lunatixz @basrieter I'd like to remove this plugin, and sorry for the delay on this. Is there a document I can follow? The original content creators have moved this context to their YouTube channel so this plugin is now not necessary with the YouTube plugin. Thank you so much for your time with this! Sorry for the ping.

**Comment by basrieter** [comment]
Just submit a new (higher) version in which you   mark it as broken in the addon.xml. See https://kodi.wiki/view/Addon.xml#%3Cbroken%3E

**Pull request merged**
